### PR TITLE
Adapt lookup init

### PR DESCRIPTION
**Pull request by denis-anisimov**

### CODE REVIEW

File: vaadin-spring/src/main/java/com/vaadin/flow/spring/VaadinApplicationConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import org.springframework.context.ApplicationContextAware;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.vaadin.flow.spring.SpringLookupInitializer.SpringApplicationContextInit;
+
+/**
+ * Vaadin Application Spring configuration.
+ * <p>
+ * Registers a default {@link ApplicationConfigurationFactory} for Vaadin web
```
Comment (mshabarov) [inline vaadin-spring/src/main/java/com/vaadin/flow/spring/VaadinApplicationConfiguration.java:27]: `ApplicationConfigurationFactory` does not exist in 12.4.

Comment (denis-anisimov) [reply vaadin-spring/src/main/java/com/vaadin/flow/spring/VaadinApplicationConfiguration.java]: Done

File: vaadin-spring/src/main/java/com/vaadin/flow/spring/VaadinServletContextInitializer.java [hunk 2]
```diff
@@ -397,12 +412,6 @@ public void contextInitialized(ServletContextEvent event) {
                     || !config.enableDevServer()) {
                 return;
             }
-            Map<String, TaskExecutor> executors = appContext
```
Comment (mshabarov) [inline vaadin-spring/src/main/java/com/vaadin/flow/spring/VaadinServletContextInitializer.java:400]: Why is this removed?

Comment (denis-anisimov) [reply vaadin-spring/src/main/java/com/vaadin/flow/spring/VaadinServletContextInitializer.java:400]: Because it's not needed (not used anywhere)

**Review by mshabarov — APPROVED** [review]

**Review by mshabarov — APPROVED** [review]
